infra/terraform/modules/data_access/teams/tests/test_create_team_bucket.py

```diff
@@ -1,10 +1,10 @@
-import json
-
 import pytest
 
 import teams
 
-from tests.conftest import TEST_TEAM_SLUG, TEST_BUCKET_NAME, TEST_BUCKET_REGION
+from tests.conftest import (TEST_TEAM_SLUG, TEST_BUCKET_NAME,
+                            TEST_BUCKET_REGION, TEST_LOGS_BUCKET_NAME,
+                            TEST_LOGS_PREFIX)
```

Comment:
And not indent to an arbitrary column - break after the open bracket and indent one level